packages/terra-clinical-data-grid/src/terra-dev-site/doc/clinical-data-grid/DatagridExamples.2/DatagridWithSubsections.scss

```diff
@@ -0,0 +1,5 @@
+:local {
+  .data-grid-div-style {
```

Comment:
We can create a common css for this one. as the height value is common across most of the examples.